lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -255,4 +310,24 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
   defp assign_form(socket, %Ecto.Changeset{} = changeset) do
     assign(socket, form: to_form(changeset))
   end
+
+  defp assign_dockerfile(socket) do
+    form = socket.assigns.dockerfile_form
+
+    version = to_string(Application.spec(:livebook, :vsn))
+    version = if version =~ "dev", do: "edge", else: version
+    apps_path = if value = form[:apps_path].value, do: value, else: "/apps"
+
+    assign(socket, :dockerfile, """
+    FROM livebook/livebook:#{version}
+
+    ENV LIVEBOOK_APPS_PATH "#{apps_path}"
+    ENV LIVEBOOK_APPS_PATH_HUB_ID "#{socket.assigns.hub.id}"
+    ENV LIVEBOOK_TEAMS_KEY "#{socket.assigns.hub.teams_key}"
```

Comment:
@josevalim it depends on their infrastructure, but ideally the image would not include the key, they would pass it when starting the container (e.g. loaded from secrets store). Even if they hardcode the key in some docker-compose/whatever config it's better than having it in the image. Doesn't sound like much hassle to me, or is there a concern I'm missing?

Reply:
I like that a lot, because we no longer pass them side by side:

```suggestion
```

And then we add a line below that says set `LIVEBOOK_TEAMS_KEY` as an environment variable for the deployment.

Reply:
Alternatively, we can add this right before CMD:

```
RUN if [ -z "$LIVEBOOK_TEAMS_KEY" ]; then echo 'Environment variable LIVEBOOK_TEAMS_KEY must be specified. Exiting.'; exit 1; fi
```

Reply:
@josevalim it needs to be a part of `CMD`, but yeah. We could also handle it on Livebook side, raise if `LIVEBOOK_TEAMS_NAME` is set, but `LIVEBOOK_TEAMS_KEY` is not.

Reply:
Ah, so let's handle it inside the Livebook code where we handle those env vars. Good call.